### PR TITLE
username.json file added

### PR DESCRIPTION
**Pull request by Shashankzz0007**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2306"></a>

### CODE REVIEW

File: data/Shashankzz0007.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Shashank Deshpande",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast! | DevOps | MERN Developer | Self-paced Full-Stack Developer",
+  "avatar": "https://github.com/Shashankzz0007.png",
+  "tags": ["Open Source", "javascript", "typescript", "Reactjs", "DevOps"],
```
Comment (Panquesito7) [inline data/Shashankzz0007.json:7]: Not sure tags can contain spaces or CAPS.
```suggestion
  "tags": ["opensource", "javascript", "typescript", "reactjs", "devops"],
```

File: data/Shashankzz0007.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/shashankz007" },
+    { "platform": "github", "url": "https://github.com/Shashankzz0007" },
+    {
+      "platform": "linkedin",
```
Comment (Panquesito7) [inline data/Shashankzz0007.json:12]: Is LinkedIn permitted in the socials field? 🤔

File: data/Shashankzz0007.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on ShowCase",
+      "url": "https://www.showwcase.com/shashankzz0007",
+      "icon": "showcase"
+    },
+    {
+      "name": "Follow me on Discord",
+      "url": "https://discord.com/channels/@me",
```
Comment (Panquesito7) [inline data/Shashankzz0007.json:43]: I don't think this will work. Also, Discord doesn't permit follows, I guess?

**Review by Panquesito7 — COMMENTED** [review]
Just a few non-blocking stuff. 🙂

File: data/Shashankzz0007.json (hunk 2 above)
Comment (Shashankzz0007) [reply data/Shashankzz0007.json:12]: @Panquesito7, I think maybe it considers as social platform but if you are suggesting I will remove it.

File: data/Shashankzz0007.json (hunk 3 above)
Comment (Shashankzz0007) [reply data/Shashankzz0007.json:43]: Thanks for pointing, I'll change that.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by Shashankzz0007** [comment]
Thank you for suggesting and merging the changes.

**Pull request merged**